### PR TITLE
fixed nav and added light theme

### PR DESCRIPTION
**Pull request by poornima-alugubelly**
Added a responsive nav
Added light theme

### CODE REVIEW

**Review by tanveertkd — APPROVED** [review]
This is good to go! 👍

File: components/List/List.html [hunk 1]
```diff
@@ -284,12 +284,8 @@ <h3>You might be interested in this</h3>
 				></small
 			>
 		</footer>
-		<script src="../../doc-page-js/Header.js"></script>
+		
 		<script src="../../doc-page-js/Sidebar.js"></script>
-		<section class="padding-s" style="width: 30rem">
```
Comment (pritam-kr) [inline components/List/List.html:289]: remove inline CSS

File: components/Navigation/Navigation.js [hunk 2]
```diff
@@ -10,6 +12,22 @@ function dropDownHandler() {
...
+
+function closeDrawer() {
+	const currDrawer = this.closest(".drawer");
+
+	currDrawer.classList.remove("active");
+}
+
+function openDrawer() {
+	const currDrawer = document.querySelector(this.dataset.drawerTarget);
+
```
Comment (pritam-kr) [inline components/Navigation/Navigation.js:24]: you can remove the gap between code line

File: components/Grid/Grid.html [hunk 3]
```diff
@@ -341,8 +341,9 @@ <h2 class="card-title">
 				></small
 			>
 		</footer>
-		<script src="../../doc-page-js/Header.js"></script>
+
 		<script src="../../doc-page-js/Sidebar.js"></script>
+		<script src="../Navigation/Navigation.js"></script>
 
 		<div></div>
```
Comment (pritam-kr) [inline components/Grid/Grid.html]: remove empty dev.

Comment (pritam-kr) [inline components/Grid/Grid.html:348]: remove empty dev.

**Review by pritam-kr — APPROVED** [review]
Good to go.

**Review by AbhishekkGautam — APPROVED** [review]
Good to go ✅

**Review by percius47 — APPROVED** [review]

**Pull request merged**